agent-bridge-datastore/src/main/java/com/newrelic/agent/bridge/datastore/JdbcHelper.java

```diff
@@ -42,27 +42,42 @@ protected Boolean initialValue() {
 
     /**
      * Any values in the urlToFactory or urlToDatabaseName Maps older than this value will
-     * be evicted from the cache when the {@link ExpiringValueCache} timer fires. This
+     * be evicted from the cache when the {@link ExpiringValueMap} timer fires. This
      * will also be used as the timer interval value.
      */
```

Comment:
I didn't catch this on the first review, but it is likely that this will be used by most instances of the map.
So maybe a static factory for this would make this easier to use.